docs/blog/2023-06-14-new-client-interface.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Mithril client has got a brand new interface
+authors:
+- name: Mithril Team
+tags: [client, certificate, mithril-stake-distribution]
+---
+
+## Mithril client now access signed Mithril Stake Distributions
+
+Since the last few months, the Mithril team has been working on signing other artifacts than just Cardano nodes' snapshots. The first other signed artifact is the stake distribution involved in Mithril Signatures. The Aggregator nodes now advertise for different types of messages to be signed and the Signer nodes know if they can handle them or not without making old nodes to crash. If you are curious, you may go and see in [Mithril Explorer](https://mithril.network/explorer), there are now different tabs, one for each type of signed artifact.
+
+The last part of this development was to make the signers aware of different kinds of artifacts and be able to verify theirs signature. This implied changing completely the command line API of the client command. By example:
```

Comment:
I think the goal of this post is to warn users that the client interface will change shortly and that the way they retrieve snapshots has changed. This is not very clear in the sub title and the below introduction

```suggestion
### Mithril client interface is evolving

For the last few months, we have implemented the capability for the Mithril protocol to sign multiple types of data: on top of the already existing Cardano node database snapshots, the Mithril stake distribution is now also signed on its own. 

In order to make the client able to work on the different types of data that are certified, we have changed its command line API.

For example:
```